index.js

```diff
@@ -71,14 +79,33 @@ function sanitize(target, options) {
     };
   });
 
-  return target;
+  return {
+    isSanitized,
+    target
+  }
+}
+
+function sanitize(target, options) {
+  return _sanitize(target, options).target;
 }
 
-function middleware(options) {
+/**
+ * @param {{replaceWith?: string, onSanitize?: function, dryRun?: boolean}} options
+ * @returns {function}
+ */
+function middleware(options = {}) {
```

Comment:
just use default parameter instead of `options = options || {};`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters